content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -30,6 +30,10 @@ Supported test frameworks:
 
 [Install the Datadog Agent to collect tests data][4].
 
+<div class="alert alert-warning">
+Agentless mode is in beta. To test this feature, follow the <a href="/continuous_integration/setup_tests/javascript#agentless-beta">instructions</a> on this page.
```

Comment:
It looks like you can add relative links into the hyperlink of the banner, which I'm surprised to learn about today!

```suggestion
Agentless mode is in beta. To test this feature, follow the <a href="/continuous_integration/setup_tests/javascript#agentless-beta">Agentless instructions</a> on this page.
```